tests/conftest.py

```diff
@@ -81,3 +83,9 @@ def mocked_pg_secret():
             Name="/copilot/dbt-app/staging/secrets/POSTGRES",
             SecretString='{"password":"abc123","dbname":"main","engine":"postgres","port":5432,"dbInstanceIdentifier":"dbt-app-staging-addons-postgresdbinstance-blah","host":"dbt-app-staging-addons-postgresdbinstance-blah.whatever.eu-west-2.rds.amazonaws.com","username":"postgres"}',
         )
+
+
+def switch_to_tmp_dir_and_copy_config_file(tmp_path, valid_config_file, destination_filename):
```

Comment:
Shall we default the `destination_filename` to `bootstrap.yml` so we don't need to pass it in each time?

```suggestion
def switch_to_tmp_dir_and_copy_config_file(tmp_path, valid_config_file, destination_filename = "bootstrap.yml"):
```